Assets/Scripts/UI/Views/Popups/MainMenu.cs

```diff
@@ -30,8 +28,15 @@ private void OnDisable()
 
         private void LoadLevel()
         {
-            _gameRun.Run(GameRunType.High);
+            var view = _uiRoot.Open<HighStoneChooseMenu>();
+            view.Closed += OnHighStoneChooseMenuClosed;
```

Comment:
Pls remove subscription and add Close()

Reply:
Saving this variable is unnecessary, just open this menu (removing "var view = ") and add Close() method then

Reply:
Done